### DISCUSSION TITLE
Pull Request Desynchronized From Fork

### DISCUSSION THREAD

**Discussion by Kurt-von-Laven** [community/community#62435] [Pull Requests]
### Select Topic Area

Bug

### Body

[Forking Renovate](https://github.com/apps/forking-renovate) opened a pull request from a fork. I [amended the commit](https://github.com/renovate-bot/pre-commit-hooks/commit/cd934de4d0c04c9ade7a0e8b75a8e37cb7c46987) to get it to pass CI and force-pushed to [the head branch of the fork](https://github.com/ScribeMD/pre-commit-hooks/compare/main...renovate-bot:pre-commit-hooks:renovate/python-3.x), but the PR (ScribeMD/pre-commit-hooks#269) doesn't reflect the update and consequently fails CI. I worked around the issue by pushing an unnecessary change and then undoing it, but I thought it might be helpful to report this nonetheless.

**Comment by mehulrajdev** [comment]
Draft pull requests are available in public repositories with GitHub Free for organizations and legacy per-repository billing plans, and in public and private repositories with GitHub Team, GitHub Enterprise Server, and GitHub Enterprise Cloud.
